### ISSUE TITLE
BARGE 2019 NLHE tournament structure

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#5]
After #4 is implemented, code up the BARGE 2019 NLHE tournament structure, and then we'll have an actual tourney that can be played while I add draw support.

**Comment by ctm** [comment]
I wound up implementing MONTE and TESLA before I got around to doing this one. Since NLHE tourneys are regularly dealt on the internet, this is now a fairly low priority enhancement, especially since it now has #30 as a prerequisite. OTOH, after #30 is done, doing this ticket should be trivial and will probably be done in conjunction with #30 since it's a test case.

**Comment by ctm** [comment]
Done and deployed, although not tested.
